src/PiWeb.Import.Sdk/Modules/ImportFormat/StandardImportFormatConfiguration.cs

```diff
@@ -11,9 +11,10 @@
 namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
 
 /// <summary>
-/// Specifies the configuration settings that are supported by the import format and default values for them.
+/// <inheritdoc />
+/// This implementation allows path rules and attribute mapping and starts with an empty mapping rules table.
 /// </summary>
-public class ImportFormatConfiguration
+public class StandardImportFormatConfiguration : IImportFormatConfiguration

```

Comment:
Is there a reason this class is called `StandardImportFormatConfiguration` but there is in contrast a `DefaultAutomationConfiguration`?

Reply:
They behave slightly different. One is a singleton while the other can be modified during creation. Maybe we can find a ways to make this more similar.

Reply:
Ok, we made the decision to rename StandardImportFormatConfiguration to ImportFormatConfiguration.